src/mscorlib/Tools/BclRewriter/BclRewriter.targets

```diff
@@ -5,30 +5,32 @@
   <!-- ********************************************************************************************* -->
 
   <PropertyGroup>
-    <BclRewriterCommand Condition="'$(BclRewriterCommand)'==''">$(CoreCLRToolsDir)\BclRewriter.exe</BclRewriterCommand>
+    <IntermediateOutputRootPath Condition="'$(IntermediateOutputRootPath)' == ''">$(IntermediateOutputPath)</IntermediateOutputRootPath>
```

Comment:
I think you should put IntermediateOutputRootPath and BuildToolsSempahore in the root dir.props as they aren't really bclrewriter specific.